field.go

```diff
@@ -1,6 +0,0 @@
-package main
```

Comment:
oh it just assume that when value is null it uses the same field name. It still supports the nested field syntax even right now 
```
{ source: "name"} 
```
is still valid as implemented.